tests/Application/public/index.php

```diff
@@ -6,8 +6,12 @@
 use Symfony\Component\HttpFoundation\Request;
 use Tests\ThreeBRS\SortingPlugin\Kernel;
 
-require dirname(__DIR__) . '/config/bootstrap.php';
+use function Symfony\Component\DependencyInjection\Loader\Configurator\env;
```

Comment:
Remove the `env` use and the `//die` part, those are remnants of our tests during call